integration-tests/src/test/java/oracle/weblogic/kubernetes/ItRemoteConsole.java

```diff
@@ -161,10 +161,17 @@ void testWlsRemoteConsoleConnectionThroughTraefik() {
         "Could not get the default external service node port");
     logger.info("Found the Traefik service nodePort {0}", traefikNodePort);
     logger.info("The K8S_NODEPORT_HOST is {0}", K8S_NODEPORT_HOST);
-    String curlCmd = "curl -v --user weblogic:welcome1 -H Content-Type:application/json -d "
+    /*String curlCmd = "curl -v --user weblogic:welcome1 -H Content-Type:application/json -d "
```

Comment:
No commented code

Reply:
Yup, that is why I haven't requested for review :) Thanks Pani!